graylog2-web-interface/src/views/components/contexts/SearchPageLayoutContext.tsx

```diff
@@ -14,19 +14,52 @@
  * along with this program. If not, see
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
+
 import * as React from 'react';
 
-type SearchPageLayoutConfig = {
-  sidebar: {
-    isPinned: boolean,
-  },
+import { singleton } from 'logic/singleton';
+
+export interface ViewActions {
```

Comment:
One note for the future: If we want to make this interface extensible through plugins, we should move this interface to `src/views/types.ts`. That way it would be decoupled from the component, making it easier to use declaration merging through module augmentation.

For now we can leave it in here though, and do this only when we encounter the actual need for it be extended through plugins.